src/ipc/ipc3/helper.c

```diff
@@ -388,6 +388,7 @@ static void comp_specific_builder(struct sof_ipc_comp *comp,
 	case SOF_COMP_DCBLOCK:
 	case SOF_COMP_SMART_AMP:
 	case SOF_COMP_CODEC_ADAPTOR:
+	case SOF_COMP_IGO_NR:
```

Comment:
You should be able to use SOF_COMP_NONE here as a general ID. It should be here so seems like a bug.

Reply:
Thanks. This is also working. I've updated the commits.